.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+
+  "rules": {
+    // Maybe enable?
+    "one-var": 0,
```

Comment:
Personally strongly prefer multiple vars for reduced diff noise/easier refactoring, but don't care much